src/main/scala/org/scalawag/jibe/Main.scala

```diff
@@ -63,7 +63,7 @@ object Main {
       val resultsDir = new File("results") / date
       val results = Executive.apply(orderedMandate, ssh, UbuntuCommander, resultsDir / "raw")
 
-      Reporter.generate(resultsDir / "raw", resultsDir / "html" / "index.html")
+      Reporter.generate(resultsDir / "raw", resultsDir / "html" / "index.html", resultsDir.getParentFile / "latest.html")
```

Comment:
I think it would be better to change what generate() takes as an argument to be "results" directory.  Then, it can derive the paths that it needs to from there.  So that it knows which run you're trying to format, it could take a second argument that's the run directory (or whatever we want to call it -- right now it's "resultsDir"). Of course, then it's not clear if "latest" should always be created.

I can see a few possible strategies:
- Only create "latest" if the date stamp being generated is indeed the latest.
- Always make "latest" point to the latest one, no matter which one you're generating.
- Always make "latest" point to the one being created, assuming it will never be called on an old one somehow.

There's another option that, instead of passing in the date directory to create a HTML report for, it could just generate any missing ones and make the "latest" link point to the newest one.  That would send a clearer message that the Reporter is in charge of the whole "results" directory (if that's a message we want to send).

Yet another: have the generate() call do what it did before and have another Reporter call that manages the "latest" link, which can always do the right thing (link to the latest, independent of report generation.

What do you guys think?